src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/xlf/MicrosoftMaintainabilityAnalyzersResources.de.xlf

```diff
@@ -127,6 +127,21 @@
         <target state="new">{0} calls {1} but does not use the value the method returns. Because {1} is marked as a Pure method, it cannot have side effects. Use the result in a conditional statement, assign the result to a variable, or pass it as an argument to another method.</target>
         <note />
       </trans-unit>
+      <trans-unit id="UseNameOfInPlaceOfStringDescription">
+        <source>Use nameof in place of string description</source>
+        <target state="new">Use nameof in place of string description</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="UseNameOfInPlaceOfStringMessage">
+        <source>Use nameof in place of string literal</source>
```

Comment:
Diagnostic message should generally be more specific to the diagnostic instance than the title. Probably something along the lines of `Use nameof in place of string literal '{0}'`